front/src/modules/ui/button/components/DropdownButton.tsx

```diff
@@ -42,7 +42,7 @@ const StyledButton = styled.button<ButtonProps & { isOpen: boolean }>`
   }
 `;
 
-const StyledDropdownItem = styled.button<ButtonProps>`
+export const StyledDropdownItem = styled.button<ButtonProps>`
```

Comment:
Maybe we should use the components in ui/dropdown/components ?

Reply:
Yes, we should!

Reply:
Yes, we should! This one will likely be removed

Reply:
it seems that you are actually using the good one in your code so you don't need to export this one